tests/ansible_collections/roles/add-custom-repos/main.yml

```diff
@@ -8,5 +8,3 @@
   # "8.6" is to be moved to the condition below related to enabling RHEL 8 EUS repos
 - import_playbook: rhel8-repos.yml
   when: ansible_facts['distribution_version'] == "8.5" or ansible_facts['distribution_version'] == "8.7"
-- import_playbook: rhel8-eus-repos.yml
-  when: ansible_facts['distribution_version'] == "8.4"
```

Comment:
I think this part should include the 8.6 instead of just removing the rhel8-eus-repos, right? 

```yaml
- import_playbook: rhel8-eus-repos.yml
  when: ansible_facts['distribution_version'] == "8.6"
```

Since 8.6 will be our lowest minor version supported for EUS, we should enable it for that version, look at this comment: https://github.com/oamg/convert2rhel/pull/808/files#diff-f3b2429fb7abe33e7f90d47b80809bf722ae4459862b257f57138761a4d03140R8

@danmyway my reasoning here is correct?

Reply:
You're right @r0x0d, I will put it back

Reply:
Thanks for noticing